userinterface/driveui.ui

```diff
@@ -39,6 +39,23 @@
      </property>
     </spacer>
    </item>
+   <item>
+    <widget class="QPushButton" name="requestRebootButton">
+     <property name="text">
+      <string>Reboot Onboard Computer</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QPushButton" name="requestShutdownButton">
+     <property name="text">
+      <string>Shutdown Onboard Computer</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLineEdit" name="requestRebootOrShutdownStatus"/>
```

Comment:
If it only outputs info, it should be a Label. QLineEdit is for inputs as well (therefore editable).

I would prefer the order: StatusLabel, RebootButton, ShutdownButton. All should be in a Group Box (e.g., "Onboard Computer" then you can drop naming it on the buttons). The group and the "Vehicle Parameters" button should move up then (above the spacer) to have a clear distinction between settings and control (manual and autopilot).

![image](https://user-images.githubusercontent.com/2404625/220844543-30b3b3af-d977-4aa4-951f-a944c6e1ba37.png)

Reply:
Agree